Examples/Physics_applications/QED/inputs.2d

```diff
@@ -57,6 +61,10 @@ electrons.uz_m = 500.
 electrons.ux_th = 2.
 electrons.uy_th = 2.
 electrons.uz_th = 50.
+##########QED####################
+electrons.do_qed = 1
```

Comment:
The idea here is to have a flag to switch qed effects on/off for a given species.
This behaviour could be changed anyway (e.g. it could make sense to have a flag to switch off all QED effects for all the species)